app/templates/account/profile.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
```

Comment:
make sure this whole section has proper formatting. It looks like you are using 8 spaces as an indent, it should be 4